ALI/ALI.py

```diff
@@ -23,6 +24,7 @@
 from CondaSetUp import CondaSetUpCall,CondaSetUpCallWsl
 import time
 import threading
+import sys
 
 
 def check_lib_installed(lib_name, required_version=None):
```

Comment:
I guess I overlooked this before since `pkg_resources` was not imported till now. `pkg_resources` is deprecated and should not be used.

You could perform the same task with `importlib.metadata`, something like:

```py
import importlib.metadata

...

installed_version = importlib.metadata.version(lib_name)
```